### PR TITLE
Release notes for AFM 1.15

### PR DESCRIPTION
**Pull request by rddheath**

### CODE REVIEW

File: modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc [hunk 1]
```diff
@@ -3,5 +3,40 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
+These release notes cover the following versions of this product:
+
+* <<1-15,1.15>>
+* <<1-1,1.1>>
+* <<1-0,1.0>>
+
+== API Functional Montitoring 1.15
```
Comment (george-hoffer) [inline modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc]: Typo:  "Montitoring"

File: modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc [hunk 2]
```diff
@@ -3,5 +3,40 @@ ifndef::env-site,env-github[]
...
+*June 15, 2019*
+
+This release introduces one new feature: you can now get notifications of failed tests through email. Select *Email* as the reporter in API Functional Monitoring in Anypoint Monitoring, or set up email as a reporter in the BAT CLI.
+
+See xref:api-functional-monitoring::afm-create-monitor.adoc[Create a Monitor] for instructions about using API Functional Monitoring in Anypoint monitoring, and see xref:api-functional-monitoring::bat-reporting-task.adoc[To Configure and View Test Reports] for details about creating reporters in the BAT CLI.
```
Comment (george-hoffer) [inline modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc]: Typo: "Anypoint monitoring" should be "Anypoint Monitoring"

File: modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc [hunk 3]
```diff
@@ -3,5 +3,40 @@ ifndef::env-site,env-github[]
...
+
+== API Functional Montitoring 1.1
```
Comment (george-hoffer) [inline modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc]: Should put a blank line after the heading. Asciidoc may have an issue trying to force these lines together without a blank line separator.

File: modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc [hunk 4]
```diff
@@ -3,5 +3,40 @@ ifndef::env-site,env-github[]
...
+
+== API Functional Monitoring 1.0
+
+This release introduces a new user interface that is incorporated into Anypoint Monitoring. To access this user interface, select *Functional Monitoring* after opening Anypoint Monitoring.
+
+For documentation about this new user interface, see xref:api-functional-monitoring::afm-in-anypoint-platform.adoc[API Functional Monitoring in Anypoint Monitoring].
+
+You can still create tests and schedule them using the BDD Test-Writing Language and the Blackbox Automated Testing (BAT) CLI. The documentation for doing that is located at xref:api-functional-monitoring::bat-top.adoc[API Functional Monitoring with the Blackbox Automated Testing (BAT) CLI].
+
+
```
Comment (george-hoffer) [inline modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc]: Should remove the following links to the docs you're deleting.

File: modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc (hunk 1 above)
Comment (rddheath) [reply modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc]: Fixed.

File: modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc (hunk 2 above)
Comment (rddheath) [reply modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc]: Fixed.

File: modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc (hunk 3 above)
Comment (rddheath) [reply modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc]: Actually, I put the date immediately after in three other sets of release notes, and asciidoc renders both headings and dates correctly.

File: modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc (hunk 4 above)
Comment (rddheath) [reply modules/ROOT/pages/api-functional-monitoring/api-functional-monitoring-release-notes.adoc]: @george-hoffer , done. Thanks!

**Pull request merged**